tensorflow/BUILD

```diff
@@ -70,6 +70,22 @@ config_setting(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
+    name = "debug",
+    values = {
+        "compilation_mode": "dbg",
+    },
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
+    name = "optimize",
```

Comment:
Haha, I hesitated myself too long on this one, that 'd' probably appeared/disappeared more than once in my editor... no problem, I'll do all recommended fixes tonight.

For your previous question, no I did not notice a crash, I was just not able to break in the C++ code before I realized that all symbols were missing.